### ISSUE TITLE
Eliminating Wind Move key bindings + Org-mode interoperability

### ISSUE THREAD

**Issue by sof5sof** [PhilipDaniels/windows-defaults#2]
The following are Org-mode's default key bindings (as found on my installation):

| command         | key     |
|-----------------|---------|
| *org-metaup*    | `M-up`  |
| *org-metadown*  | `M-down`|
| *org-metaleft*  | `M-left`|
| *org-metaright* |`M-right`|

with *org-meta<arrow>* delegating calls to:
- *org-move-subtree-(arrow)*
- *org-table-move-row*
- *org-move-item-(arrow)*

`M-(arrow)` is my personal currently preferred key binding for manipulating Org objects.

And, at least under the Emacs-style configuration of key bindings in Spacemacs, they are currently unshadowed by the Org layer (or any other Configuration Layer that I am currently using).

This isn't to say that it shouldn't be acceptable for a Configuration Layer to use unoccupied key bindings for its various functions!

However **my main point** is that I would rather not have *windmove-(arrow)*  functions be assigned at all under this Configuration Layer.

This is especially important now when @syl20bnr has given a green light to start integrating it, see [Spacemacs Issue #5429](https://github.com/syl20bnr/spacemacs/issues/5429#issuecomment-397783821):
> PR welcome to integration @PhilipDaniels work

To my observation, window manipulations are already covered in Spacemacs via the following key bindings:

| command             | key          |
|---------------------|--------------|
| *evil-window-up*    | `(Leader)-k` |
| *evil-window-down*  | `(Leader)-j` |
| *evil-window-right* | `(Leader)-l` |
| *evil-window-left*  | `(Leader)-h` |
 
**In conclusion I propose the removal** of the following which is currently implemented in this *window-defaults* layer:

| command             | key          |
|---------------------|--------------|
| *windmove-(arrow)*    | `M-(arrow)` |

P.S.

For me, I would like to see this layer integrated (or some version of it) as I dabble with many other text editing environments which all mostly support Windows-style key bindings (at least to some extent).

This would make interoperability with other environments much easier (less cognitive load when context switching).

Needless to say this would also make some Spacemacs users enjoy a more approachable editing environment.

Best regards
